standalone-metastore/metastore-server/src/main/java/org/apache/hadoop/hive/metastore/RawStore.java

```diff
@@ -1572,9 +1579,20 @@ void getFileMetadataByExpr(List<Long> fileIds, FileMetadataExprType type, byte[]
    * @return list of primary key columns or an empty list if the table does not have a primary key
    * @throws MetaException error accessing the RDBMS
    */
+  @Deprecated
   List<SQLPrimaryKey> getPrimaryKeys(String catName, String db_name, String tbl_name)
       throws MetaException;
 
+  /**
+   * Get the primary associated with a table.  Strangely enough each SQLPrimaryKey is actually a
```

Comment:
"Get the primary key/s associated with a table." Please do not write personal opinion in the comments, unless they are purely technical. You can simply say "SQLPrimaryKey represents a single primary key column. Since a table can have one or more primary keys ( in case of composite primary key ), this method returns List< SQLPrimaryKey>".

Reply:
Right. I will keep in mind.